crates/turbopack-build/src/chunking_context.rs

```diff
@@ -22,12 +24,37 @@ use crate::ecmascript::node::{
     chunk::EcmascriptBuildNodeChunk, entry::chunk::EcmascriptBuildNodeEntryChunk,
 };
 
+#[derive(
+    Serialize,
+    Deserialize,
+    Debug,
+    Clone,
+    Copy,
+    Hash,
+    PartialEq,
+    Eq,
+    PartialOrd,
+    Ord,
+    TraceRawVcs,
+    Default,
+)]
+pub enum MinifyType {
```

Comment:
Should this just be a boolean? Are we expecting more variants?

Reply:
afaik bool wouldn't allow a `true` default value, and this is more self-documenting.

Reply:
Agreed, use enums instead of booleans where it makes more sense.